### PR TITLE
net-im/libcommuni: bump to version 3.5.0_p20190110

### PR DESCRIPTION
**Pull request by ConiKost**
Package-Manager: Portage-2.3.76, Repoman-2.3.17
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>

### CODE REVIEW

File: net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qml test +uchardet"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	uchardet? ( app-i18n/uchardet )
+	!uchardet? ( dev-libs/icu:= )
+	"
```
Comment (juippis) [inline net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild]: Mis-indent (doesn't this show as red in vim?)

File: net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+EGIT_COMMIT="88bb0fcdaab5bdd1176a3b6b485a6415a137ec27"
+
+inherit qmake-utils
+
+DESCRIPTION="A cross-platform IRC framework written with Qt"
+HOMEPAGE="http://communi.github.io/"
```
[the remark this answers is not in the document]
Comment (ConiKost) [reply net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild]: Fixed.

File: net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild (hunk 1 above)
Comment (ConiKost) [reply net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild]: Nope, doesn't show as red here. Thanks, fixed.

File: net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	# Currently the test tst_IrcLagTimer fails, so disabling
+	# See: https://github.com/communi/libcommuni/issues/63
+	sed -e '/irclagtimer/d' -i tests/auto/auto.pro || die
+}
+
+src_configure() {
+	local myqmakeargs=( libcommuni.pro
+		# Disables compile run-time benchmarks, as they don't make any sense
+		-config no_benchmarks
+		# Disables examples, as no new files are installed
```
[the remark this answers is not in the document]
Comment (ConiKost) [reply net-im/libcommuni/libcommuni-3.5.0_p20190110.ebuild]: Good catch :-) The examples are indeed build, but not installed by themself. Changed ebuild to handle this. Thanks!

**Pull request closed without merging**